python/MoltresXS.py

```diff
@@ -0,0 +1,280 @@
+"""
+     Python Function that takes homoginezed cross sections and prepares them for MOLTRES
+
+"""
+
+import io
+import sys
+import numpy as np
+import h5py
+from pyne import serpent
+
+
+class scale_xs:
+    """
+        Python class that reads in a scale t16 file and organizes the cross section data
+        into a numpy dictionary. Currently set up to read an arbitrary number of energy groups,
+        an arbitrart number of delayed neutron groups, an arbitrary number of identities,
+        an arbitrary number of temperature branches, and an arbitrary number of burnups.
+
+        Parameters
+        ----------
+        FileIn: str
```

Comment:
So, MOOSE uses crazy non-pep8 Python, but we really don't have to. But, in any case, this is neither [pep8-compatible](https://www.python.org/dev/peps/pep-0008/) variable naming nor [MOOSE-style-guide-compatible](https://mooseframework.inl.gov/old/wiki/CodeStandards/) variable naming.

Also, a more specific name is always better.

```suggestion
        xs_filename: str
```